saleor/plugins/webhook/utils.py

```diff
@@ -104,3 +106,9 @@ def parse_payment_action_response(
             "transaction_already_processed", False
         ),
     )
+
+
+@contextmanager
+def _catch_duration_time():
```

Comment:
This function is not "private", since you don't use it in this module, but it is imported in some other place.